### ISSUE TITLE
asks for password when opening xps file

### ISSUE THREAD

**Issue by GoogleCodeExporter** [JustLoy77/ebookdroid#550]
```
What steps will reproduce the problem?
1. navigate to xps file 
2. select file opens ebookdroid 
3.

What is the expected output?
xps file use to open 

What do you see instead?
asks for password and the file is not password protected 

What version of the product are you using?
The latest from the play store 


On what operating system?


What device are you using?


Please provide any additional information below.

```

Original issue reported on code.google.com by `exit6...@gmail.com` on 21 Jul 2013 at 12:58

**Comment by GoogleCodeExporter** [comment]
```
I forgot to add the following information. 
Ebookdroid version 2.0.0
Operating system 4.1.2 android 
Galaxy S3


```

Original comment by `exit6...@gmail.com` on 21 Jul 2013 at 1:33

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 2.0.1
```

Original comment by `Alexander.V.Kasatkin@gmail.com` on 21 Jul 2013 at 8:55
- Changed state: **Fixed**
